Xamarin.Forms.DataGrid/DataGrid.xaml.cs

```diff
@@ -86,11 +86,17 @@ public partial class DataGrid : Grid
 			BindableProperty.Create(nameof(IsSortable), typeof(bool), typeof(DataGrid), true);
 
 		public static readonly BindableProperty FontSizeProperty =
-			BindableProperty.Create(nameof(FontSizeProperty), typeof(double), typeof(DataGrid), 13.0);
+			BindableProperty.Create(nameof(FontSize), typeof(double), typeof(DataGrid), 13.0);
 
 		public static readonly BindableProperty HeaderFontSizeProperty =
 			BindableProperty.Create(nameof(HeaderFontSize), typeof(double), typeof(DataGrid), 13.0);
 
+		public static readonly BindableProperty FontFamilyProperty =
+			BindableProperty.Create(nameof(FontFamily), typeof(string), typeof(DataGrid), Font.Default.FontFamily);
+
+		public static readonly BindableProperty HeaderFontFamilyProperty =
```

Comment:
Since I've Added `HeaderLabelStyle` this is not needed. I am also planning to obsolete `HeaderFontSize` & `HeaderTextColor` properties. Could you remove this?